Shamaz/SecondViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  SecondViewController.swift
+//  Shamaz
+//
+//  Created by Alexandra Ivanova on 31/03/2020.
+//  Copyright © 2020 Alex Ivanova. All rights reserved.
+//
+
+import UIKit
+
+@available(iOS 13.0, *)
+class SecondViewController: UIViewController {
+
+    @IBOutlet weak var nextPlayerButton: UIButton!
+    @IBOutlet weak var playerTaskLabel: UILabel!
+    
+    var recievedPhrase = "Player's task"
+    var nextPlayerPhrase = ""
+    var nextPlayerNumber = Int.random(in: 1...10)
+    
+    let design = Design()
```

Comment:
Alternatively, you could use `static` or `Singleton`

Reply:
I'll use static. Thnx :)